### ISSUE TITLE
Chisels and bits with ArchitectureCraft - ElytraDev Version Crash

### ISSUE THREAD

**Issue by Lt1489** [Lunatrius/Schematica#298]
I am running a custom pack, when I try to load in a schematic, it crashes.
lunatriuscore 1.12.2-1.2.0.42
schematica 1.12.2-1.8.0.167

Crash Log:
https://pastebin.com/CUSSXAMi

**Comment by Lt1489** [comment]
Update: Setting it to structure file format (nbt) works

**Comment by Lt1489** [comment]
I believe the issue is due to chisels and bits

**Comment by Lunatrius** [comment]
Does this happen every time you try to load the schematic?

**Comment by Lt1489** [comment]
No, only on the build that I was working on, I'm unsure why, but when I removed the chisels and bits windows I made, it doesn't crash when I save the schematic and load it. There are still other chisels and bits in the build tho, so I'm unsure why the windows with the build caused an crash.

**Comment by Lunatrius** [comment]
It crashed because something else was modifying the tile entity list when the renderer was trying to get the active tile entity.
